docs/UpgradeGuide.md

```diff
@@ -84,7 +84,7 @@ Datadog.configure do |c|
   # Global settings
   c.tracer.hostname = '127.0.0.1'
   c.tracer.port = 8126
-  c.runtime_metrics.enabled = true
+  c.runtime_metrics_enabled = true
```

Comment:
👋 @ivanbrennan, thanks for reaching out!

The UpgradeGuide is actually correct today.
You can see that `runtime_metrics.enabled` is indeed separated into two parts now:
https://github.com/DataDog/dd-trace-rb/blob/a349a7cd19c4881213365f5827b304c3518e800e/spec/datadog/core/configuration/settings_spec.rb#L506

Are you have any issues setting `c.runtime_metrics.enabled = ...` in your application?

Reply:
The line I changed is supposed to show the old configuration. It's part of an example that shows an old config, followed by its equivalent new config.

Reply:
Ha, thank you for pointing that out!